### PR TITLE
Pull Request Resolving Issue #5166

### PR DESCRIPTION
**Pull request by nshah25**
Hello, I am working on issue #5166 and have added //to_lua and //end_lua tags in the Statistics.h file and updated the APIDesc.lua file with the changes made. This is my first pull request, so please let me know if I need to do anything else! Thanks!

### CODE REVIEW

**Comment by 12xx12** [comment]
Hi and welcome to the project.
Thank you for the PR!

A small tip reguarding the checkstyle: Set up you IDE to use tabs as indentation.
I think you'll need to exportt the statistic enum to the lua API also to acess the values.

File: Server/Plugins/APIDump/APIDesc.lua [hunk 1]
```diff
@@ -12162,6 +12162,72 @@ end
...
+			]],
+			Functions = 
+			{
+				SetValue =
+				{
+					Params =
+					{
+						{
+							Name = "a_Stat",
+							Type = "<unknown>"
```
Comment (12xx12) [inline Server/Plugins/APIDump/APIDesc.lua:12179]: This should be the enum value. Look at the other enums in the APIDesc file

File: Server/Plugins/APIDump/APIDesc.lua [hunk 2]
```diff
@@ -12162,6 +12162,72 @@ end
...
+						}
+					
+						{
+							Name = "a_Value",
+							Type = "<unknown>"
```
Comment (12xx12) [inline Server/Plugins/APIDump/APIDesc.lua:12184]:
```suggestion
							Type = "number"
```

File: Server/Plugins/APIDump/APIDesc.lua [hunk 3]
```diff
@@ -12162,6 +12162,72 @@ end
...
+						},
+					}
+						
+					Returns =
+					{
+						Type = "void",	
+					},
```
Comment (12xx12) [inline Server/Plugins/APIDump/APIDesc.lua:12191]:
```suggestion
				
```

File: Server/Plugins/APIDump/APIDesc.lua [hunk 4]
```diff
@@ -12162,6 +12162,72 @@ end
...
+					Params = 
+					{
+						{
+							Name = "a_Stat",
+							Type = "<unknown>"
+						},
+						{
+							Name = "a_Delta",
+							Type = "<unknown>"
+						}
```
Comment (12xx12) [inline Server/Plugins/APIDump/APIDesc.lua:12205]: same here

File: Server/Plugins/APIDump/APIDesc.lua [hunk 5]
```diff
@@ -12162,6 +12162,72 @@ end
...
+							Type = "<unknown>"
+						},
+						{
+							Name = "a_Delta",
+							Type = "<unknown>"
+						}	
+					},
+					Returns = 
+					{
+						Type = "StatValue",
```
Comment (12xx12) [inline Server/Plugins/APIDump/APIDesc.lua:12209]:
```suggestion
						Type = "number",
```

**Review by 12xx12 — CHANGES_REQUESTED** [review]

**Comment by nshah25** [comment]
Thank you so much! This was really helpful and my partner and I are both really excited to help contribute to Cuberite!

**Pull request closed without merging**